prelude/endianness.md

```diff
@@ -0,0 +1,19 @@
+The endianness defines how integers and pointers are encoded and decoded.
+
+```rust
+pub enum Endianness {
+    LittleEndian,
+    BigEndian,
+}
+
+pub use Endianness::*;
+
+impl Endianness {
+    /// If `signed == Signed`, the data is interpreted as two's complement.
+    pub fn decode(self, signed: Signedness, bytes: List<u8>) -> BigInt { todo!() }
```

Comment:
This is written as `todo!()` instead of `;` for now, so that things still compile.